roles/cobbler_profile/tasks/import_distro.yml

```diff
@@ -0,0 +1,88 @@
+---
+# This profile will do all the work necessary to create a new distro/profile
+# pair in Cobbler.
+
+# Since this profile will be used several times in the same playbook,
+# mention the distro name each time.
+- name: Distro name
+  debug: var=distro_name
+
+- name: Load extra_distros from secrets
+  set_fact:
+      distros: "{{ distros.update(extra_distros|default({})) }}{{ distros}}"
```

Comment:
I can add a comment:

The jinja2 templating API allows you to update one dict with another, but it does so in-place without returning any dict. The first blob does the updating; the second actually returns the result.